_field-types/supported-field-types/match-only-text.md

```diff
@@ -0,0 +1,101 @@
+        "type" :  "match_only_text"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Parameters
+
+While `match_only_text` supports most parameters available for text fields, modifying most of them can be counterproductive. This field type thrives on simplicity and efficiency, minimizing data stored in the index to optimize storage costs. Therefore, keeping the default settings is generally the best approach. Any adjustments beyond analyzer settings can reintroduce overhead and negate the efficiency benefits of `match_only_text`.
+
+The following table lists all parameters available for `match_text_only` fields.
+
+Parameter | Description
+:--- | :---
+`analyzer` | The analyzer to be used for this field. By default, it will be used at index time and at search time. To override it at search time, set the `search_analyzer` parameter. Default is the `standard` analyzer, which uses grammar-based tokenization and is based on the [Unicode Text Segmentation](https://unicode.org/reports/tr29/) algorithm.
+`boost` |  All hits are assigned a score of 1 and are multiplied by `boost` to produce the final score for the query clause.
+`eager_global_ordinals` | Specifies whether global ordinals should be loaded eagerly on refresh. If the field is often used for aggregations, this parameter should be set to `true`. Default is `false`.
+`fielddata` | A Boolean value that specifies whether to access analyzed tokens for this field for sorting, aggregation, and scripting. Default is `false`.
+`fielddata_frequency_filter` | A JSON object that specifies to load into memory only those analyzed tokens whose document frequency is between the `min` and `max` values (provided as either an absolute number or a percentage). Frequency is computed per segment. Parameters: `min`, `max`, `min_segment_size`. Default is to load all analyzed tokens.
+`fields` | To index the same string in several ways (for example, as a keyword and text), provide the fields parameter. You can specify one version of the field to be used for search and another to be used for sorting and aggregations.
+`index` | A Boolean value that specifies whether the field should be searchable. Default is `true`.
+`index_options` | You cannot modify this parameter.
+`index_phrases` | Not supported.
+`index_prefixes` | Not supported.
+`meta` | Accepts metadata for this field.
+`norms` | Norms are disabled and cannot be enabled.
+`position_increment_gap` | Although positions are disabled, `position_increment_gap` behaves similarly to the text field for phrase queries. Such queries may be slower but are still functional.
+`similarity` | Setting similarity has no impact. The `match_only_text` field type doesn't support queries like `more_like_this`, which rely on similarity. Use a `keyword` or `text` field for queries that rely on similarity.
+`term_vector` | Term vectors are supported but using them is discouraged because it contradicts the primary purpose of this field---storage optimization.
```

Comment:
```suggestion
`term_vector` | Term vectors are supported, but using them is discouraged because it contradicts the primary purpose of this field---storage optimization.
```